docs/topics/dynamic-content.rst

```diff
@@ -248,8 +248,34 @@ Using a headless browser
 A `headless browser`_ is a special web browser that provides an API for
 automation.
 
-The easiest way to use a headless browser with Scrapy is to use Selenium_,
-along with `scrapy-selenium`_ for seamless integration.
+Since version 2.0, it is possible to integrate libraries that use the
+``async/await`` syntax. One such library is `pyppeteer`_ (an unnoficial
+Python port of `puppeteer`_), which uses headless Chrome to download and
+render pages.
+The following is a simple snippet to illustrate its usage within Scrapy::
+
+    import pyppeteer
+    import scrapy
+
+    class PyppeteerSpider(scrapy.Spider):
+        name = "pyppeteer"
+        start_urls = ["data:,"]  # avoid making an actual upstream request
+
+        async def parse(self, response):
+            browser = await pyppeteer.launch()
+            page = await browser.newPage()
+            await page.goto("https:/example.org")
+            title = await page.title()
+            yield {"title": title}
```

Comment:
I think this needs `await browser.close()` to be correct, probably with some sort of try/except, and avoid leaking resources.